public/app/plugins/sdk.ts

```diff
@@ -1,7 +1,12 @@
-import { PanelCtrl } from 'app/features/panel/panel_ctrl';
-import { MetricsPanelCtrl } from 'app/features/panel/metrics_panel_ctrl';
-import { QueryCtrl } from 'app/features/panel/query_ctrl';
-import { alertTab } from 'app/features/alerting/AlertTabCtrl';
+import { makeClassES5Compatible } from '@grafana/data';
 import { loadPluginCss } from '@grafana/runtime';
+import { PanelCtrl as PanelCtrlES6 } from 'app/features/panel/panel_ctrl';
+import { MetricsPanelCtrl as MetricsPanelCtrlES6 } from 'app/features/panel/metrics_panel_ctrl';
+import { QueryCtrl as QueryCtrlES6 } from 'app/features/panel/query_ctrl';
+import { alertTab } from 'app/features/alerting/AlertTabCtrl';
+
+const PanelCtrl = makeClassES5Compatible(PanelCtrlES6);
+const MetricsPanelCtrl = makeClassES5Compatible(MetricsPanelCtrlES6);
+const QueryCtrl = makeClassES5Compatible(QueryCtrlES6);
```

Comment:
Why @ts-ignore is not necessary here? Or is it?

Reply:
because these are exported to plugins via systemjs and we have not types for these